src/Hubbup.Web/Program.cs

```diff
@@ -1,13 +1,26 @@
+using System.Threading.Tasks;
+using Hubbup.Web.DataSources;
 using Microsoft.AspNetCore;
 using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
 
 namespace Hubbup.Web
 {
     public class Program
     {
-        public static void Main(string[] args)
+        public static async Task Main(string[] args)
         {
-            CreateWebHostBuilder(args).Build().Run();
+            var host = CreateWebHostBuilder(args).Build();
+
+            // Load data before we start things up.
+            var logger = host.Services.GetRequiredService<ILogger<Program>>();
+            logger.LogDebug("Loading repo set and person set data...");
+            var dataSource = host.Services.GetRequiredService<IDataSource>();
+            await dataSource.ReloadAsync(default);
```

Comment:
This is the pattern we're recommending in the EF initialization scenarios and it works well here too.